api/src/wfl/service/google/bigquery.clj

```diff
@@ -46,29 +46,35 @@
       util/response-body-json
       :tables))
 
-(defn query-table-sync
-  "Query for a BigQuery TABLE within a Data Repo SNAPSHOT in
-   Google Cloud PROJECT synchronously. Using non-legacy query
-   SQL syntax. Note: table and views are interchangeable BigQuery
-   concepts here, so both of them work as `TABLE`.
-
-   At least `data custodian` permission on the snapshot is
-   required for running the query job.
+(defn query-sync
+  "Given QUERY, look for rows in a BigQuery table within a
+   Google Cloud PROJECT synchronously, using non-legacy query
+   SQL syntax. Return flatten rows.
 
    Parameters
    ----------
    project  - Google Cloud Project to list the BigQuery datasets in.
-   snapshot - Data Repo Snapshot.
-   table    - BigQuery table/view name."
-  [project snapshot table]
-  (let [query (format "SELECT * FROM `%s.%s.%s`" project snapshot table)]
-    (-> (str/join "/" ["projects" project "queries"])
-        bigquery-url
-        (http/post {:headers (auth/get-auth-header)
-                    :body    (json/write-str
-                              {:query query
-                               :use_legacy_sql false})})
-        util/response-body-json)))
+   query    - BigQuery Standard SQL query string."
+  [project query]
+  (-> (str/join "/" ["projects" project "queries"])
+      bigquery-url
+      (http/post {:headers (auth/get-auth-header)
+                  :body    (json/write-str
+                            {:query          query
+                             :use_legacy_sql false})})
+      util/response-body-json))
+
+;; visible for testing
+(defn parse-row
+  "Parse row information from BigQuery `rows` entry in query response."
+  [rows]
+  (map :v (:f rows)))
```

Comment:
Does this need to be public now? I think it's better to `flatten-rows` at the end of `query-sync` and hide the `:f` `:v` nonsense.

Reply:
I thought your "flatten-rows" just removes the `:f` and the `:v`. We'd probably always want to do that?

Reply:
done this